frontend/test/model/SchemaTest.scala

```diff
@@ -0,0 +1,25 @@
+package model
+
+import model.RichEvent.GuLiveEvent
+import play.api.test.PlaySpecification
+import play.api.libs.json.{Json}
+import model.EventbriteTestObjects._
+
+class SchemaTest extends PlaySpecification {
+
+  "EventSchema" should {
+
+    "generate JSON-LD schema data for an event" in {
+
+      val event = GuLiveEvent(eventWithName("Test Event"), None, None)
+      val json = Json.toJson(event.schema)
+
+      (json \ "@context").as[String] === "http://schema.org"
```

Comment:
Yeah I think the implementation of `EventSchema.from(this)` feels a bit strange to me. You could just pass it the fields it needs in each place?